plugins/woocommerce/client/legacy/js/flexslider/jquery.flexslider.js

```diff
@@ -824,16 +810,17 @@
             return (posCalc * ((slider.vars.rtl)?1:-1)) + "px";
           }());
 
+      dur = (dur !== undefined) ? (dur/1000) + "s" : "0s";
+      slider.container.css("transition-duration", dur);
+
       if (slider.transitions) {
         target = (vertical) ? "translate3d(0," + target + ",0)" : "translate3d(" + (parseInt(target)+'px') + ",0,0)";
-        dur = (dur !== undefined) ? (dur/1000) + "s" : "0s";
-         slider.container.css("transition-duration", dur);
+      } else {
+        slider.container.css("transition-timing-funciton", easing);
```

Comment:
I believe there is a small typo in here it should be `transition-timing-function`, right?

```suggestion
        slider.container.css("transition-timing-function", easing);
```

Reply:
Fix typo.